.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you unignored it, every change will be seen as modified in git. it would be nice to have ide-agnostic solution, but i guess editorconfig is not enough